src/components/TreeSelect/TreeSelect.stories.tsx

```diff
@@ -3,34 +3,34 @@ import { TreeSelect } from "components/TreeSelect";
 
 const treeData = [
```

Comment:
Same question about having `Components/` in the `title` of `TreeSelect`

<img width="424" alt="Screen Shot 2022-06-29 at 1 20 43 PM" src="https://user-images.githubusercontent.com/47064971/176497751-ef943fe9-f1f7-40e2-86f0-984cef768bfa.png">